src/App.js

```diff
@@ -1,25 +1,33 @@
-import logo from './logo.svg';
-import './App.css';
+// src/Greeting.js
+import React, { useEffect, useState } from 'react';
+
+const Greeting = () => {
+  const [greeting, setGreeting] = useState('');
+
+  useEffect(() => {
+    const fetchGreeting = async () => {
+      try {
+        const response = await fetch('http://localhost:3001/random_greeting');
+        const data = await response.json();
+        setGreeting(data.greeting); // Assuming the API response has a 'greeting' property
+      } catch (error) {
+        console.log(error);
+      }
+    };
+
+    // Call the fetchGreeting function
+    fetchGreeting();
+  }, []); // The empty dependency array ensures that the effect runs only once
```

Comment:
- [ ] Excellent progress on your implementation, and I appreciate your responsiveness to the previous reviewer's feedback! 🚀 However, there are still some important aspects that require attention, specifically related to the project requirements.

   - As outlined in the project specifications, the use of `Redux` for state management is strongly recommended. Although I noticed that you've set up the `reducer`, `action`, and `store`, it appears that you're not leveraging these structures as intended but rather the `API` data is being handled directly in your `useEffect` currently. Below is a screenshot of the project requirement:
   
<img width="910" alt="Screenshot 2023-12-19 at 08 13 07" src="https://github.com/SaveryIV/hello-react-front-end/assets/107192076/2f38846d-f34b-4515-9e6a-6e46a4773354">


   - To align with the project's guidelines, I recommend revisiting your implementation and integrating `Redux` more prominently. Utilize the store to manage and distribute the data throughout your frontend application. This adjustment will not only ensure compliance with the specified requirements but also enhance the scalability and maintainability of your codebase. 

Happy coding! 😊👨‍💻🚀

Reply:
I think im using the correct guidelines now, please check and let me know